dashboard/src/pages/StudyConfPage/forms/general/General.tsx

```diff
@@ -134,21 +82,45 @@ const General: React.FC<Props> = ({ id, localData, confKeys }: Props) => {
     'general'
   );
 
-  const onSubmit: SubmitHandler<FormData> = formData => {
+  const onSubmit = (e: any): void => {
+    e.preventDefault();
+
     const data = {
       [id]: formData,
     };
 
     createStudyConf({ data, studySlug });
   };
 
+  const validateInput = (name: string, value: any) => {
```

Comment:
Hm. Can we use type information to do this? Is there a more general solution to this problem?

Reply:
I see what you mean so create some kind of validator that says if this field is of type `number` run this validation otherwise give me the raw value.